<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuCliente</class>
 <widget class="QDialog" name="MenuCliente">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Cliente</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>39</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Nome</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>150</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>130</y>
     <width>431</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>71</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cartão de</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEmail">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>420</y>
     <width>421</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>390</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Email</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtontRes">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>490</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Residencial</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCelular">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>520</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineLogradouro">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>300</y>
     <width>421</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCredito">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>421</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineRg">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>210</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CEP</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>460</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Telefones</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCidade">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTelDDI">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>520</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Logradouro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>10</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>RG</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCelularDDI">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>520</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTel">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>520</y>
     <width>121</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCep">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>240</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Celular</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>331</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCpf">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>40</y>
     <width>181</width>
     <height>30</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Cidade</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineTelDDD">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonCom">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>107</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Comercial</string>
   </property>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>151</width>
     <height>541</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="botaoCadastrar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Cadastrar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoNovo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Novo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoBuscar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Buscar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoEditar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoExcluir">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Excluir</string>
    </property>
   </widget>
   <widget class="QPushButton" name="botaoSair">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>460</y>
      <width>131</width>
      <height>71</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Sair</string>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineNome">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>421</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Bairro</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Crédito</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineCelularDDD">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>520</y>
     <width>31</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>61</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>CPF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineBairro">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>180</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineNumero">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>180</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineCpf</tabstop>
  <tabstop>lineRg</tabstop>
  <tabstop>lineNome</tabstop>
  <tabstop>lineBairro</tabstop>
  <tabstop>lineNumero</tabstop>
  <tabstop>lineCidade</tabstop>
  <tabstop>lineCep</tabstop>
  <tabstop>lineLogradouro</tabstop>
  <tabstop>lineCredito</tabstop>
  <tabstop>lineEmail</tabstop>
  <tabstop>lineCelularDDI</tabstop>
  <tabstop>lineCelularDDD</tabstop>
  <tabstop>lineCelular</tabstop>
  <tabstop>lineTelDDI</tabstop>
  <tabstop>lineTel</tabstop>
  <tabstop>lineTelDDD</tabstop>
  <tabstop>radioButtontRes</tabstop>
  <tabstop>radioButtonCom</tabstop>
  <tabstop>pushButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
